### PR TITLE
blockcache

### PR DESCRIPTION
**Pull request by ellemouton**

### CODE REVIEW

File: blockcache/blockcache.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+		Cache: lru.NewCache(capactity),
+	}
+}
+
+func (bc *BlockCache) GetBlock(hash *chainhash.Hash,
+	getBlockImpl func(hash *chainhash.Hash) (*wire.MsgBlock,
+		error)) (*wire.MsgBlock, error) {
+
+	// Create an inv vector for getting the block.
+	inv := wire.NewInvVect(wire.InvTypeWitnessBlock, hash)
```
Comment (ellemouton) [inline blockcache/blockcache.go:26]: not sure if it is fine to hard code this...

Comment (carlaKC) [reply blockcache/blockcache.go:26]: Should be fine, do we hard code it elsewhere in the codebase?

**Review by carlaKC — COMMENTED** [review]
Do you still need me to take a look at this?

**Comment by ellemouton** [comment]
@carlaKC , nope. all good 👍 can remove yourself as reviewer if you want :) I forgot that you are probably getting spammed with update emails from this 😅

**Pull request closed without merging**